react/training-project/src/app/screens/Game/index.js

```diff
@@ -27,33 +27,28 @@ class Game extends Component {
       );
     });
 
-    return (
-      <div className={style.game}>
-        <div className={style.gameBoard}>
-          <Board squares={current.squares} />
-        </div>
-        <div className={style.gameInfo}>
-          <h3 className={style.status}>
-            {winner ? `Winner: ${winner}` : `Next Player ${this.props.xIsNext ? 'X' : 'O'}`}
-          </h3>
-          <ol>{moves}</ol>
-        </div>
-      </div>
+    return this.props.logged ? (
+      <GameDumb current={current} winner={winner} xIsNext={this.props.xIsNext} moves={moves} />
+    ) : (
+      <Redirect to="/login" />
```

Comment:
Have a constants file in the routes folder and add this one there. Do it in another PR.